sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -235,6 +235,19 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * Construct a `DataFrame` representing the database table accessible via JDBC URL
    * url named table and connection properties.
    *
+   * The `table` parameter can be based on a derived table, which is generated within the scope of
+   * a query FROM clause. For example, you could use this subquery as `table:
+   * {{{
+   *    "(SELECT * FROM customers WHERE ...) AS tmp"
+   * }}}
```

Comment:
This duplicates `docs/sql-programming-guide.md`. I don't think we have to add these.

Reply:
Perfect! This is a part of the documentation, that I seam to have missed.